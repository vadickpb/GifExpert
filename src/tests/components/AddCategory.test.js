const { shallow } = require("enzyme")
const { AddCategory } = require("../../components/AddCategory")
import React from 'react';
describe('Pruebas en el componente AddCategory', () => {
    
    const setCategories = () =>{}
    const wrapper = shallow(<AddCategory setCategories = {setCategories} />)

    test('Debe mostrar el wrapper', () => {
        expect(wrapper).toMatchSnapshot();
    })

    // test('debe cambiar la caja de texto', () => {
    //     const input = wrapper.find('input');
    //     const value = 'hola mundo';
    //     input.simulate('change', {target : { value }})
        
    //     expect(wrapper.find('p').text().trim()).toBe(value);
    // })
    
    
})
