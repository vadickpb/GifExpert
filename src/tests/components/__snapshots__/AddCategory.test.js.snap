// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en el componente AddCategory Debe mostrar el wrapper 1`] = `
<form
  onSubmit={[Function]}
>
  <p />
  <input
    onChange={[Function]}
    type="text"
    value=""
  />
</form>
`;
